File: android/src/main/kotlin/io/devcrew/snapchatloginkit/model/TokenResponse.kt
package io.devcrew.snapchatloginkit.model

class TokenResponse(override var code: Int = 200, override var message: String = "Success") : SnapchatResponse {
    var token: String? = null
    override fun toMap(): Map<String, Any?> = mapOf(
        "code" to code,
        "message" to message,
        "token" to token
    )
}

File: ios/Classes/Model/TokenResponse.swift
//
//  TokenResponse.swift
//  integration_test
//
//  Created by Najam Us Saqib on 26/02/2024.
//

import Foundation

// MARK: - Token Response -

struct TokenResponse: SnapchatResponse {
    var code: Int = 200
    var message: String = ""
    var token: String?
    
    func toMap() -> [String : Any?] {
        return [
            "code" : code,
            "message" : message,
            "token" : token
        ]
    }
}
